cmd/bridge/main.go

```diff
@@ -53,6 +53,9 @@ const (
 
 	// Well-known location of metering service for OpenShift. This is only accessible in-cluster.
 	openshiftMeteringHost = "reporting-operator.openshift-metering.svc:8080"
+
+	// Well-known location of the GitOps service. This is only accessible in-cluster
+	openshiftGitOpsBackendHost = "cluster.openshift-pipelines-app-delivery.svc:8080"
```

Comment:
nit: "Backend" is implied and none of the other variables has it in the name

```suggestion
	openshiftGitOpsHost = "cluster.openshift-pipelines-app-delivery.svc:8080"
```